### ISSUE TITLE
PNG lenth not enough in timeline

### ISSUE THREAD

**Issue by classenu** [jliljebl/flowblade-forum#13]
In V1, there is a video that is 30 minutes long. In V2, I put a PNG image used for titles background. 
The problem is that, I can't set the PNG length to 30 minutes. The largest length value is 00:04:14:04(frame 7624). If I set the frame to 15000, a strange operational problems started to appear: unable to adjust the PNG clip length by dragging it with the mouse.

My version:
Flowblade 2.0.0 (Flathub) / Debian stretch 4.9.0.9  with gnome

**Comment by jliljebl** [comment]
Yes, this is a current limitation caused by MLT library used by Flowblade. This can change in the future but for now the suggested workaround is to use multiple PNGs to get the length desired.

I will look at exact problem described in post to see if there is something that can be done.

**Comment by PanderMusubi** [comment]
I've noticed that setting the length greater than 7500 frames also has some strange side effects, e.g. blend down has difficulties. Reported for version 2.6.3. So using a max of 7500 frames and copying that PNG a few times is a workaround, but very cumbersome. Also setting the cache to 2 GB helps to maintain GUI responsiveness when using PNG that last thousands of frames.

This is also a laborious workaround for https://github.com/jliljebl/flowblade/issues/975

**Comment by PanderMusubi** [comment]
Some info on the limitation in MLT library is found at https://github.com/mltframework/mlt/pull/264#discussion_r144201769

Currently, these files in MLT work with this value:
- src/framework/mlt_producer.h
- src/framework/mlt_producer.c
- src/modules/opengl/filter_movit_white_balance.yml
- src/modules/avformat/producer_avformat.c

As they discussed in 2017, it could be swapped for MAX_INT, perhaps for interoperability purposes, better set to e.g. 16,777,216.

@jliljebl shall we create an issue at MLT or even a PR?
